### PR TITLE
PLAT-476: BargeIn leak in SMDelegatedTokenRequest

### PR DESCRIPTION
**Pull request by bigbes**

### CODE REVIEW

File: ledger-core/conveyor/smachine/slot_aliases.go [hunk 1]
```diff
@@ -55,13 +55,14 @@ func (a *slotAliases) RetainAll(parent *slotAliases, removeFn func(k interface{}
 			}
 		}
 	}
+
+	keys := a.keys[i:]
```
Comment (sexyhamster) [inline ledger-core/conveyor/smachine/slot_aliases.go:59]: ????

Comment (bigbes) [reply ledger-core/conveyor/smachine/slot_aliases.go:59]: !!!!

Comment (bigbes) [reply ledger-core/conveyor/smachine/slot_aliases.go:59]: There was a bug in a part where cleanup was processed, so it's @cyraxred fix for that problem

Comment (cyraxred) [reply ledger-core/conveyor/smachine/slot_aliases.go:59]: it is ;)

**Pull request merged**